### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by JfffV**

### CODE REVIEW

**Comment by discrimine** [comment]
To decrease reviewing files, always push empty react application before first PR

File: public/logo192.png
Comment (discrimine) [inline public/logo192.png:1]: If this is a commercial project, and you gonna change the fav icon, do not push the default one

File: public/logo512.png
Comment (discrimine) [inline public/logo512.png:1]: The same here

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Comment (discrimine) [inline public/manifest.json]: Change to the real one

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.App {
+  text-align: center;
```
Comment (discrimine) [inline src/App.css:2]: Is this doing smth? I don't think you have this class in the component
also nit: extra lines at the end of the file

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import AboutUs from "./components/AboutUs";
+import { useState } from "react";
+
+function App() {
+  return (
+    <div className="main">
+      <BrowserRouter>
+        <Header />
+        <Routes>
+          <Route path="blogs" element={<Body />} />
```
Comment (discrimine) [inline src/App.js]: Rename body component to blogs or smth more specific

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+          <Route path="about" element={<AboutUs />} />
+        </Routes>
+        <Footer />
+      </BrowserRouter>
+
```
Comment (discrimine) [inline src/App.js]: nit: extra line

File: src/components/AboutUs.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./About_us.css";
+
+export default function AboutUs() {
+  return (
+    <footer>
```
Comment (discrimine) [inline src/components/AboutUs.js]: Footer is a part of your main component, you shouldn't use it in all other components

File: src/components/Body.css [hunk 6]
```diff
@@ -0,0 +1,14 @@
+.presentation{
```
Comment (discrimine) [inline src/components/Body.css:1]: Formatting:
space before open bracket,
tabs in .title class

Use auto formatter

File: src/components/Body.css
Comment (discrimine) [inline src/components/Body.css:1]: Nit: inconsistent empty lines
I would suggest to have 1 empty line after each close bracket, and emty lines between logical groups of props in a case of 100500 props classes

File: src/components/About_us.css
Comment (discrimine) [inline src/components/About_us.css:1]: Inconsistent name, component camelCase, css file snake_case

File: src/components/Footer.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
```
Comment (discrimine) [inline src/components/Footer.js:1]: Formatting

File: src/components/Header.js
Comment (discrimine) [inline src/components/Header.js:1]: Formatting

File: src/img/homepage.jpg
Comment (discrimine) [inline src/img/homepage.jpg:1]: Duplicated image, I saw it at the begining of the pr in another folder

File: src/components/Our_works.js
Comment (discrimine) [inline src/components/Our_works.js:1]: Use camelCase as component name

File: src/logo.svg
Comment (discrimine) [inline src/logo.svg:1]: Do you need the default react logo?

File: src/components/AboutUs.js (hunk 5 above)
Comment (JfffV) [reply src/components/AboutUs.js]: @discrimine asd

**Review by discrimine — APPROVED** [review]

**Pull request merged**